src/main/scala/io/phdata/retirementage/loadgen/LoadGenerator.scala

```diff
@@ -167,13 +170,13 @@ object LoadGenerator {
 
     // Create a temporary dimension dataframe with the incorrect column names
```

Comment:
what do you mean by 'incorrect column names'?

Reply:
It's incorrect column names because it grabs the dimensionid from the parent table, however, I want there column dimensionid to be named id so it can seamlessly be used as a parent to another dimension table